packages/ketcher-react/src/script/editor/shared/closest.ts

```diff
@@ -386,7 +420,8 @@ function findClosestRgroupAttachmentPoints(
   let closestItemId;
 
   restruct.visibleRGroupAttachmentPoints.forEach((reItem, id) => {
-    const itemOutlinePoints = reItem.getOutlinePoints();
+    // Remove readonly from getOutlinePoints return type
+    const itemOutlinePoints = [...reItem.getOutlinePoints()];
```

Comment:
Why do we need this?

Reply:
We return readonly array but the function expects just an array, we need to copy it.
I can update the comment if it's not clear